docs/source/routing/performance/caching/entity.mdx

```diff
@@ -135,6 +215,54 @@ This entry contains an object with the `all` field to affect all subgraph reques
 
 ### Entity cache invalidation
 
+```mermaid
+%%{init: {"flowchart": {"htmlLabels": false}} }%%
+flowchart RL
+	subgraph QueryResponse["Cache invalidation POST"]
+		n1["{
+			&emsp;&emsp;&quot;kind&quot;: &quot;subgraph&quot;,
+			&emsp;&emsp;&quot;subgraph&quot;: &quot;price&quot;,
+			&emsp;&emsp;&quot;type&quot;: &quot;Price&quot;,
+			&emsp;&emsp;&quot;key&quot;: {
+			&emsp;&emsp;&emsp;&emsp;&quot;id&quot;: &quot;101&quot;
+			&emsp;&emsp;}
+		}"]
+    end
+
+	subgraph Subgraphs["Subgraphs"]
+		Purchases["Purchases"]
+		Inventory["Inventory"]
+		Price["Price"]
+	end
+
+	subgraph PriceQueryFragment["Price Query Fragment (e.g. TTL 2200)"]
+		n2["` ̶{̶
```

Comment:
Similar to the above, I found that Mermaid didn't support the delete sequence in Markdown, so I kludged together this solution for crossing out the individual characters.